src/vsr/replica.zig

```diff
@@ -4637,8 +4637,8 @@ pub fn ReplicaType(
                 }
                 break :commit_min_min min;
             };
-            assert(commit_min_min <= self.commit_min);
-            const commit_lag = self.commit_min - commit_min_min;
+            assert(commit_min_min <= self.commit_min + constants.pipeline_prepare_queue_max);
```

Comment:
Would be great to add a comment in code explaining! I guess that’s an old primary who committed ahead of the newer primary? I wonder if that’s an interesting replica_test scenario?